### DISCUSSION TITLE
I must be missing it — where is the version number in the UI?

### DISCUSSION THREAD

**Discussion by p-jones** [alexjustesen/speedtest-tracker#367] [Q&A] [answered]
Sorry for the very basic question.

I can see Watchtower updated me this AM to the latest Speedtest-tracker image, but where in the UI is the version exposed? 

Thanks for writing this excellent software!

**Comment by alexdelprete** [comment, marked as answer]
Homepage, left column, bottom:

![image](https://user-images.githubusercontent.com/7027842/215325577-a2d06c8d-61ba-413a-8564-56db2465f1f4.png)

**Comment by p-jones** [comment]
I knew I would feel dumb :(I just needed to scroll down.Thank you!

**Reply by alexdelprete** [reply]
no worries...it happens. :)

**Comment by dissidente** [comment]
Hi.
Are these version tags still present in the UI?
I've just updated speedtest tracker docker container and was trying to validate the update to make sure all was good, but on my main page there are no version tags shown in the UI. Using Chrome on Android (in DeX mode, but don't think this is relevant)
Is there anywhere else in the docker image where I can check the version?
Thanks in advance

**Reply by alexjustesen** [reply]
Yep, still available. If you're on a mobile device you'll want to open the nav panel and scroll to the bottom.

**Reply by dissidente** [reply]
Yeah, seems that Chrome in DeX mode doesn't scroll all the way down, so I couldn't see that section 😒
